README.md

```diff
@@ -70,6 +76,12 @@ or:
       manage_package => false,
     }
 
+Add custom crontab run-parts
+
+    class { 'cron':
```

Comment:
could you add ```puppet on top of the code and ``` and the bottom?

Reply:
Not sure I follow, what do you mean?

Reply:
@treydock I guess @bastelfreak is talking about the markdown syntax for code highlighting.  See https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code

Reply:
I edited @bastelfreak comment to render properly, should make more sense now :-)

Reply:
This change was made